pages/manage_and_operate/secret_manager/external-secret-operator/guide.en-gb.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Use Kubernetes External Secret Operator with Secret Manager"
+excerpt: "Configure External Secret Operator to store Kubernetes secrets on the OVHcloud Secret Manager"
+updated: 2025-11-07
+---
+
+> [!primary]
+> Secret Manager is currently in beta phase. This guide can be updated in the future with the advances of our teams in charge of this product.
+
+## Objective
+
+This guide explains how set up the Kubernetes External Secret Operator to use the OVHcloud Secret Manager as a provider
+
+## Requirements
+
+- An [OVHcloud customer account](/pages/account_and_service_management/account_information/ovhcloud-account-creation).
+- Have [ordered an OKMS domain](/pages/manage_and_operate/kms/quick-start) or [created a first secret](/pages/manage_and_operate/secret_manager/secret-manager-ui).
+
+## Instructions
+
+### Setup the Secret Manager
+
+To allow access to the Secret Manager you will need to create credentials.
+
+Create an [IAM local user](/pages/account_and_service_management/account_information/ovhcloud-users-management) with acces right on your domain.
```

Comment:
Is it possible to do this step though a CLI or TF ?

Because now the user needs to do several clicks and needs to fil several forms in the manager. Not easy at all.
And because we don't help him, the user can do a lot of mistakes.

It's a guide about ESO and secret manager, not IAM so the IAM step needs to be easy :) .